packages/@instana-integration/jenkins/dashboards/jobs-and-overview.json

```diff
@@ -0,0 +1,380 @@
+{
+  "writable": true,
+  "id": "vUsXSUuaSp2DyCRAS-JkRA",
+  "title": "Jenkins Jobs & Overview",
```

Comment:
Both the name for the file and the title could be changed, e.g.: overview-and-jobs will be better?